extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+- Fix a bug where persisted databases may be lost if VS code is restarted while databases are being initialized. [#1638](https://github.com/github/vscode-codeql/pull/1638)
```

Comment:
```suggestion
- Fix a bug where databases may be lost if VS Code is restarted while the extension is being started up. [#1638](https://github.com/github/vscode-codeql/pull/1638)
```

I think this describes the impact in a way the user can understand a bit better as the average user may not know when databases are being initialized.

Reply:
Thanks. That sounds better.